### PR TITLE
prepare deb packaging for dvoting without unikernel

### PR DESCRIPTION
**Pull request by jbsv**
`make deb` generates an installable, debian package.
Successfully tested on 3 local VMware machines.

### CODE REVIEW

File: deb-package/README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+### Network config
+
+Ensure that the public address is correct. For instance, in `network.env`, replace:
+```sh
+export dela_public="//localhost:9000"
+```
+with the node's public address:
+```sh
+`export dela_public="//172.16.253.150:9000"`
```
Comment (jbsv) [inline deb-package/README.md:43]: quote mess

File: deb-package/build-deb.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+# fpm needs an existing output directory
+OUTPUT_DIR="dist"
+mkdir -p $OUTPUT_DIR
+
+fpm \
+    --force -t deb -a all -s dir -C deb -n dedis-dvoting -v ${VERSION} \
+    --iteration ${ITERATION} \
+    --deb-user dvoting \
+    --deb-group dvoting \
+    --depends bridge-utils \
```
Comment (jbsv) [inline deb-package/build-deb.sh:52]: whoops

**Review by jbsv — COMMENTED** [review]
changes needed

File: deb-package/pkg/opt/dedis/dvoting/bin/start-dvoting [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+sleep 5
+
+source /opt/dedis/dvoting/config/config.env
+source /etc/dedis/dvoting/network.env
+
+echo "Starting ${dela_bin} on ${dela_listen} using folder ${dela_data} ..."
+
+pk=3c07e93b9d99032366f7d92697f8dc1337bf8bb617b3faa6389e94d18e3d0e40
```
Comment (nkcr) [inline deb-package/pkg/opt/dedis/dvoting/bin/start-dvoting:10]: this should be set on `config.env`

**Review by nkcr — COMMENTED** [review]
Nice ! Just a minor comment

**Review by nkcr — APPROVED** [review]
LGTM 🚀

**Pull request merged**